src/pages/developers/learning-tools.tsx

```diff
@@ -226,6 +228,40 @@ const LearningToolsPage = ({
     },
   ]
 
+  interface ILearningToolsCardGridProps {
+    category: Array<ILearningTool>
+  }
+
+  const LearningToolsCardGrid: React.FC<ILearningToolsCardGridProps> = ({
```

Comment:
Lets create a new file for this component

Reply:
@pettinarip Just committed a patch for this. 
Two interfaces were needed, one depending on the other, so I tried just pulling these into the `./types.ts` file to keep it more organized instead of exporting the type interface from the component file. Let me know what you think.